### PR TITLE
API Route: Driver registration

### LINKED ISSUE #18: Create API Route for Driver Registration
Create API Route for Driver Registration

### PR DESCRIPTION
**Pull request by malivinayak**
API Route for Driver Registartion
resolves #18

### CODE REVIEW

**Review by OutdatedGuy — APPROVED** [review]
~~LGTM!~~

File: src/backend/routes/registration/driverRegistration.js [hunk 1]
```diff
@@ -80,6 +77,18 @@ const driverRegistration = async (req, res) => {
                 });
             }
 
+            // Unique aadhar
+            query = `select Aadhar_Number from EMPLOYEE where Aadhar_Number = :1`;
```
Comment (OutdatedGuy) [inline src/backend/routes/registration/driverRegistration.js]: Is field name not case-sensitive? At Line 119 we are using all caps name.

**Review by OutdatedGuy — CHANGES_REQUESTED** [review]

**Comment by malivinayak** [comment]
resolves #18

**Comment by OutdatedGuy** [comment]
Also change `dbo` variable name to `birthDate`

File: src/backend/routes/registration/driverRegistration.js [hunk 2]
```diff
@@ -80,6 +77,18 @@ const driverRegistration = async (req, res) => {
                 });
             }
 
+            // Unique aadhar
+            query = `select AADHAR_NUMBER from EMPLOYEE where Aadhar_Number = :1`;
```
Comment (OutdatedGuy) [inline src/backend/routes/registration/driverRegistration.js]: Capitalize the second aadhar_number too.

File: src/backend/routes/registration/driverRegistration.js [hunk 3]
```diff
@@ -80,6 +77,18 @@ const driverRegistration = async (req, res) => {
...
+            const checkaadharNumber = await connection.execute(query, [aadharNumber], options);
+
+            if (checkaadharNumber.rows[0] !== undefined) {
+                return res.status(403).send({
+                    message: "Aadhar_Number is already registered!!!\nYou can either Register with different Aadhar_Number",
```
Comment (OutdatedGuy) [inline src/backend/routes/registration/driverRegistration.js]: Don't send `Aadhar_Number` to frontend, do `Aadhar Number`. And remove `You can either Register with different Aadhar_Number` as it doesn't make any sense

File: src/backend/routes/registration/driverRegistration.js [hunk 4]
```diff
@@ -91,6 +100,7 @@ const driverRegistration = async (req, res) => {
                     code: 403,
                 });
             }
+
             // Unique Lincence_Number
             query = `select Lincence_Number from EMPLOYEE where Lincence_Number = :1`;
```
Comment (OutdatedGuy) [inline src/backend/routes/registration/driverRegistration.js]: Capitalize the database fields

**Review by OutdatedGuy — CHANGES_REQUESTED** [review]

**Review by OutdatedGuy — APPROVED** [review]
LGTM!

**Pull request merged**